test/e2e/es_cluster_logging.go

```diff
@@ -75,7 +75,7 @@ func ClusterLevelLoggingWithElasticsearch(f *Framework) {
 
 	// Wait for the Elasticsearch pods to enter the running state.
 	By("Checking to make sure the Elasticsearch pods are running")
-	label := labels.SelectorFromSet(labels.Set(map[string]string{"name": "elasticsearch-logging"}))
+	label := labels.SelectorFromSet(labels.Set(map[string]string{"k8s-app": "elasticsearch-logging"}))
```

Comment:
Well the source of the bug is that the YAML spec was changed without finding all the places that were impacted. So a constant does not help here. I think this is a more general problem we will keep seeing because of the way we use and implement labels.